### ISSUE TITLE
Add: Emisora Comunitaria 104.1 FM SD

### ISSUE THREAD

**Issue by songwenhui239** [iptv-org/iptv#39797]
### Stream ID (required)

EmisoraComunitaria1041FM.co@SD

### Stream URL (required)

https://104florencia.co/wp-admin/admin-ajax.php?action=frp_hls_proxy&url=https%3A%2F%2Fstreamyes.alsolnet.com%2Fcomunitariafm%2Flive%2Fplaylist.m3u8

### Quality

None

### Label

None

### Contributing Guide

- [x] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md)
